tests/src/test/scala/org/http4s/multipart/MultipartSuite.scala

```diff
@@ -55,29 +57,34 @@ class MultipartSuite extends Http4sSuite {
       val field1 =
         Part.formData[IO]("field1", "Text_Field_1", `Content-Type`(MediaType.text.plain))
       val field2 = Part.formData[IO]("field2", "Text_Field_2")
-      val multipart = Multipart(Vector(field1, field2))
-      val entity = EntityEncoder[IO, Multipart[IO]].toEntity(multipart)
-      val body = entity.body
-      val request =
-        Request(method = Method.POST, uri = url, body = body, headers = multipart.headers)
-      val decoded = EntityDecoder[IO, Multipart[IO]].decode(request, true)
-      val result = decoded.value
-
-      assertIOBoolean(result.map(_ === Right(multipart)))
+      Boundary
+        .fromScalaRandom[IO](random)
+        .flatMap { boundary =>
```

Comment:
The purer API is a bit more awkward to call.  I'm open to putting a convenience on `Multipart`

Reply:
It's a little better if we make a Multiparts algebra.